src/components/Contact/Contact.jsx

```diff
@@ -0,0 +1,146 @@
+import React from "react";
+import { Row, Col } from 'reactstrap';
+import "./Contact.css";
+import { IconContext } from "react-icons";
+import { graphql, useStaticQuery } from 'gatsby';
+import Img from 'gatsby-image';
+import {
+  FaGithubSquare,
+  FaFacebookSquare,
+  FaTwitterSquare,
+  FaDiscord,
+  FaYoutubeSquare,
+  FaRedditSquare,
+  FaGooglePlusSquare,
+  FaComments
+} from "react-icons/fa";
+
+
+export default props => {
+  const data = useStaticQuery(graphql`
+    query myImages2{
+      historyImg: file(relativePath: {eq: "images/3.jpg"}){
+        childImageSharp{
+          fluid(maxWidth: 300){
+            ...GatsbyImageSharpFluid_tracedSVG
+          }
+      }
+    },
+    modding: file(relativePath: {eq: "images/9.jpg"}){
+      childImageSharp{
+        fluid(maxWidth: 300){
+          ...GatsbyImageSharpFluid
+        }
+      }
+    },
+    community: file(relativePath: {eq: "images/8.jpg"}){
+      childImageSharp{
+        fluid(maxWidth: 300){
+          ...GatsbyImageSharpFluid_tracedSVG
+        }
+      }
+    }
+  }
+`);
```

Comment:
We aren't using the queried images anywhere on the page, why query them?

Reply:
Yes. Optimized :)